test/forge/integration/LiquidateIntegrationTest.sol

```diff
@@ -74,43 +100,6 @@ contract LiquidateIntegrationTest is BaseTest {
         uint256 lltv;
     }
 
-    function testLiquidateMargin(LiquidateTestParams memory params, uint256 amountSeized, uint256 elapsed) public {
```

Comment:
Why removing this test ? It should pass independently on the changes made on `liquidate`